src/app/pages/transaction-request/components/transaction-error/transaction-error.tsx

```diff
@@ -25,6 +26,15 @@ export enum TransactionErrorReason {
 
 const TransactionErrorSuspense = memo(() => {
   const reason = useTransactionError();
+  const analytics = useAnalytics();
+
+  useEffect(() => {
+    if (!reason) return;
+    void analytics.track(
+      `transaction_signing_error_${TransactionErrorReason[reason].toLowerCase()}`
```

Comment:
I'd suggest a uniform event name here (`view_transaction_signing_error`) with the reason included as a `reason` property. That way we can track the overall volume most effectively, and slice the data further from there by querying by reason.